### ISSUE TITLE
Web interface becomes unresponsive with large amount of buckets(10000)

### ISSUE THREAD

**Issue by Monkeybanana1** [minio/minio#5536]
Web interface becomes unresponsive with a large amount of buckets. When creating 10000 buckets the web interface tries to load all buckets in the search area. This makes the interface really slow, almost not usable.

Steps to reproduce.
- Start minio server. Which way doesn't matter.
- Use script to create 10000 buckets using API or by hand.
- Open web interface and try to search for a specific bucket.

Possible solution:
Don't load all buckets in the web interface. Show standard only the first 20 buckets and make it an option to load the next 20 buckets.

**Comment by kmkanagaraj** [comment]
@Monkeybanana1 Thanks for reporting. We will fix this as part of the rewrite effort (#5409).

**Comment by deekoder** [comment]
@kmkanagaraj  can we test if this issue does not occur in the newux branch?

**Comment by kmkanagaraj** [comment]
@deekoder this is not yet fixed. It involves code re-structuring. I will fix this once @rushenn is done with the ux changes to avoid conflicts.

**Comment by kmkanagaraj** [comment]
Fixed in #5684

**Comment by kmkanagaraj** [comment]
The fix got reverted while reverting the newux changes from master. It will go in when the current newux branch is merged into master.

**Comment by deekoder** [comment]
@kmkanagaraj  any updated on this? Has this made it to the master branch?

**Comment by kmkanagaraj** [comment]
@deekoder The fix is available in newux branch, need to backport it to the master branch, will do that in future.

**Comment by harshavardhana** [comment]
closing as a duplicate of #8181
